src/BlockEntities/BeaconEntity.cpp

```diff
@@ -234,28 +234,22 @@ void cBeaconEntity::GiveEffects(void)
 
 	bool HasSecondaryEffect = (m_BeaconLevel >= 4) && (m_PrimaryEffect != m_SecondaryEffect) && (m_SecondaryEffect > 0);
 
-	Vector3d BeaconPosition(m_Pos);
-	GetWorld()->ForEachPlayer([=](cPlayer & a_Player)
+	auto Area = cBoundingBox(m_Pos, Radius, cChunkDef::Height - 1, static_cast<double>(-m_Pos.y));
```

Comment:
According to the wiki, this should actually be:

```suggestion
	auto Area = cBoundingBox(m_Pos, Radius, Radius + static_cast<double(cChunkDef::Height), -Radius);
```

That way the AABB extends above the beacon by 256 blocks, and below it by Radius.